### PR TITLE
Cleanup and working TensorflowProducer, 2022 Data Updates

### PR DESCRIPTION
**Pull request by carriganm95**
Updates to:

background calculation script
calculation of fiducial maps
2022 scale factors
L1ETM measurement scripts
adding jetvetomap and cut for jetvetomap
adding cuts for deep sets and fake tracks (0.5 threshold currently)
update luminosity of 2022 samples

### CODE REVIEW

File: BackgroundEstimation/python/config_cfg.py [hunk 1]
```diff
@@ -69,9 +69,10 @@ def getNLayersChannelVariations (chName):
 
 # "Old" style cuts (nPixel >= 3, nValidHits >= 7) t&p channels for fiducial map
 # Use these!
-if False:
```
Comment (borzari) [inline BackgroundEstimation/python/config_cfg.py:72]: I think I would keep this as `False`, to be consistent with the rest.

File: StandardAnalysis/test/test_tensorflow_MC2022_cfg.py [hunk 2]
```diff
@@ -474,3 +474,145 @@
...
+process.tensorflowProducer.maxNumOfRecHits = cms.int32(-1)
+process.tensorflowProducer.signalTriggerNames = cms.vstring([
+        'HLT_MET105_IsoTrk50_v',
+        'HLT_PFMET120_PFMHT120_IDTight_v',
+        'HLT_PFMET130_PFMHT130_IDTight_v',
+        'HLT_PFMET140_PFMHT140_IDTight_v',
+        'HLT_PFMETNoMu120_PFMHTNoMu120_IDTight_v',
+        'HLT_PFMETNoMu130_PFMHTNoMu130_IDTight_v',
+        'HLT_PFMETNoMu140_PFMHTNoMu140_IDTight_v',
+        'HLT_PFMET250_HBHECleaned_v',
```
Comment (borzari) [inline StandardAnalysis/test/test_tensorflow_MC2022_cfg.py:533]: We are not using the `HBHECleaned` paths anymore. Maybe this should be updated with the paths in the trigger config file

**Review by borzari — COMMENTED** [review]
I am a bit confused about the jet veto maps. Shouldn't they be added by default in some event selection? We will remove those regions in any case, aren't we?

File: StandardAnalysis/test/test_tensorflow_MC2022_cfg.py (hunk 2 above)
Comment (carriganm95) [reply StandardAnalysis/test/test_tensorflow_MC2022_cfg.py:533]: this is actually junk now that the OSUGenericTrack producer handles this, I would remove the tensorflow producers if we don't need it

Comment (borzari) [reply StandardAnalysis/test/test_tensorflow_MC2022_cfg.py:533]: As discussed in MM, these can be removed in this PR, then

Comment (carriganm95) [reply StandardAnalysis/test/test_tensorflow_MC2022_cfg.py]: removed

**Pull request merged**